.circleci/config.yml

```diff
@@ -721,6 +721,7 @@ jobs:
 
   backend-integration-tests-SK1:
     <<: *base-job
+    resource_class: macos.m1.medium.gen1
```

Comment:
Do we want this?

Reply:
CircleCI is getting rid of Intel in June. So good to migrate these early.